eclair-core/src/test/scala/fr/acinq/eclair/blockchain/bitcoind/BitcoinCoreClientSpec.scala

```diff
@@ -938,16 +938,20 @@ class BitcoinCoreClientSpec extends TestKitBaseClass with BitcoindService with A
     sender.expectMsg(tx1)
   }
 
-  test("compute pubkey from a receive address") {
+  test("get pubkey for p2wpkh receive address") {
```

Comment:
we could run this test with bitcoin core set to use bech32m for both receive and change addresses ?

Reply:
Done in https://github.com/ACINQ/eclair/pull/2486/commits/6bc5a1b49605eb307627af86489e61c4741a9c97